daphne/cli.py

```diff
@@ -14,6 +17,17 @@
 DEFAULT_PORT = 8000
 
 
+class ASGI3Middleware:
+    def __init__(self, app):
+        self.app = app
+
+    def __call__(self, scope):
+        return functools.partial(self.asgi, scope=scope)
```

Comment:
@tomchristie I think the middleware should change the asgi version in the scope.

Reply:
Good call.